Ensure all non-test Typescript files have < 600 lines

Sauradip07 opened the pull request:
<!--
This section can be deleted after reading.

We employ the following branching strategy to simplify the development process and to ensure that only stable code is pushed to the `master` branch:

- `develop`: For unstable code: New features and bug fixes.
- `master`: Where the stable production ready code lies. Only security related bugs.

NOTE!!!

ONLY SUBMIT PRS AGAINST OUR `DEVELOP` BRANCH. THE DEFAULT IS `MAIN`, SO YOU WILL HAVE TO MODIFY THIS BEFORE SUBMITTING YOUR PR FOR REVIEW. PRS MADE AGAINST `MAIN` WILL BE CLOSED.
-->

<!--
Thanks for submitting a pull request! Please provide enough information so that others can review your pull request.
-->

**Ensure all non-test Typescript files have < 600 lines**

<!-- E.g. a bugfix, feature, refactoring, etc… -->

**Issue Number:**#1298

Fixes #1298

**Did you add tests for your changes?**
No

**Summary**

The modified code checks TypeScript files (*.ts and *.tsx) excluding test files, ensuring they do not exceed a specified line limit.
Ensure all non-test Typescript files have < 600 lines - Admin

**Does this PR introduce a breaking change?**
No

**Other information**
muatations.ts Line Count : 669  -> 598
Queries.ts Line Count : 810  -> 575
OrgList.tsx Line Count : 647  -> 568

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

Yes

palisadoes reviewed (changes requested):
1. You need to edit this file and change the line count from 1000 to 600 as part of the PR 
    - https://github.com/PalisadoesFoundation/talawa-admin/blob/develop/.github/workflows/pull-requests.yml
1. The file `package-lock.json` must not be a part of this PR
1. You have created new Mutation and Query files. Why have you not created the equivalent test files so that life will be easier for future contributors wanting to update any future tests to the code? These should be created.

Sauradip07 commented:
@palisadoes 
The Mutation and Query files are written in the GraphQL Schema Definition Language (SDL). Writing tests for a schema file is often considered optional for several reasons. That’s why I haven’t written any test files for these, **and there were no test files for the Mutation and Query file previously**.

And I removed `package-lock.json` from the PR 
added line count from 1000 to 600 

`Please feel free to reach out if there are any modifications or any changes need to be made.`
